monGARS/api/authentication.py

```
@@ -1,16 +1,21 @@
 def get_current_user(token: str = Depends(oauth2_scheme)) -> dict:
-    sec = SecurityManager(secret_key=settings.SECRET_KEY, algorithm=settings.JWT_ALGORITHM)
+    sec = SecurityManager(
+        secret_key=settings.SECRET_KEY, algorithm=settings.JWT_ALGORITHM
+    )
     try:
         payload = sec.verify_token(token)
         return payload
     except Exception as e:
-        raise HTTPException(status_code=401, detail=f"Invalid token: {e}")
\ No newline at end of file
+        raise HTTPException(status_code=401, detail=f"Invalid token: {e}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)